apps/backoffice-v2/src/pages/Statistics/Statistics.page.tsx

```diff
@@ -1,5 +1,24 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { useZodSearchParams } from '@/common/hooks/useZodSearchParams/useZodSearchParams';
+import { DateRange, dateRangeSchema } from '@/common/hooks/useZodSearchParams/dateRangeSchema';
 
 export const Statistics: FunctionComponent = () => {
```

Comment:
If the date input would live in the home route then it also makes sense for this logic to be in the home route.